utils/common/src/main/java/org/apache/brooklyn/util/http/HttpTool.java

```diff
@@ -250,6 +254,21 @@ public static HttpClientBuilder httpClientBuilder() {
         private boolean trustAll;
         private boolean trustSelfSigned;
 
+        public static HttpClientBuilder fromBuilder(HttpClientBuilder other) {
+            HttpClientBuilder result = httpClientBuilder();
+            result.clientConnectionManager = other.clientConnectionManager;
+            result.httpParams = other.httpParams;
```

Comment:
`httpParams` is leftover code which is deprecated so it can be removed now but it is ok to remove it later as well.